src/containers/hoc/enhance-accessibility.jsx

```diff
@@ -42,6 +43,8 @@ const AccessibleHOC = (WrappedComponent) => {
     setA11yNavigatedMessage: PropTypes.func.isRequired,
   };
 
+  ScreenReaderAccessibleComponent.WrappedComponent = WrappedComponent;
```

Comment:
Shouldn't this be part of the constructor? Different instances would share the same class, so we want to set the `WrappedComponent` on the instance, not on the class.

Reply:
TIL: dont need do constructor (this doesn't work anyways in this case but can do)
`static WrappedComponent = WrappedComponent;` inside the class